controllers/ingredients.controller.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const db = require('../models').db;
+const filterProps = require('../services/utils.js').filterProps;
+
+
+class IngredientsController {
+  constructor (ingredientModel) {
+    if (!ingredientModel) throw new Error('Ingredient model not provided');
+    this.Ingredient = ingredientModel;
+    this.createIngredient = this.createIngredient.bind(this);
```

Comment:
Just FYI: if you use BABEL, you can install the plugin [transform class properties](https://babeljs.io/docs/en/babel-plugin-transform-class-properties/) to do something like

```js
createIngredient = async () => {
   …
}
```

So you won't need to bind it. It's just another syntax, nothing wrong with your approach.